src/main/play-doc/operation/CLI.md

```diff
@@ -32,7 +32,7 @@ commands:
 ...
 ```
 
-## Packaging configuration
+## Packaging Configuration
```

Comment:
I am wrong here. Based on http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html we need to capitalize `Configuration` as well.